XRTK-Core/Packages/com.xrtk.core/Definitions/Controllers/MixedRealityControllerVisualizationProfile.cs

```diff
@@ -130,7 +130,7 @@ public bool TryGetControllerPose(Type controllerType, Handedness hand, out Mixed
             {
                 if (controllerVisualizationSettings[i].ControllerType != null &&
                     controllerVisualizationSettings[i].ControllerType.Type == controllerType &&
-                    (controllerVisualizationSettings[i].Handedness == hand || controllerVisualizationSettings[i].Handedness == Handedness.Both) &&
+                    (controllerVisualizationSettings[i].Handedness == hand || controllerVisualizationSettings[i].Handedness == Handedness.Any) &&
```

Comment:
I wanna say there's a specific reason for this.

Have to dig is up and discuss this change.